src/components/ui/OakRadioButton/OakRadioButton.tsx

```diff
@@ -47,57 +48,81 @@ const HiddenRadioButtonInput = styled.input.attrs({
 `;
 
 type VisibleRadioButtonInputProps = OakFlexProps & {
+  $disableFocusRing: boolean;
+  $radioInnerSize: OakAllSpacingToken;
   disabled?: boolean;
 };
 
 const VisibleRadioButtonInput = styled(OakFlex)<VisibleRadioButtonInputProps>`
+  border-radius: 50%;
 
-  ${HiddenRadioButtonInput}:focus-visible ~ &::before {
-    content: "";
-    height: ${parseSpacing("all-spacing-7")};
-    width: ${parseSpacing("all-spacing-7")};
-    background: "transparent"
-    display: block;
-    position: absolute;
-    border-radius: 50%;
-    border: ${parseBorder("border-solid-m")} ${parseColor("grey60")};
-    box-shadow: ${`inset 0 0 0 0.13rem ${parseColor("lemon")}`};
-  }
+  ${(props) =>
+    !props.$disableFocusRing &&
+    css`
+      ${HiddenRadioButtonInput}:focus-visible ~ &::before {
+        content: "";
+        height: ${parseSpacing("all-spacing-7")};
+        width: ${parseSpacing("all-spacing-7")};
+        background: "transparent"
+        display: block;
+        position: absolute;
+        border-radius: 50%;
+        border: ${parseBorder("border-solid-m")} ${parseColor("grey60")};
+        box-shadow: ${`inset 0 0 0 0.13rem ${parseColor("lemon")}`};
+      }`}
 
   ${HiddenRadioButtonInput}:checked ~ &::after {
     content: "";
-    height: ${parseSpacing("all-spacing-4")};
-    width: ${parseSpacing("all-spacing-4")};
+    height: ${(props) => parseSpacing(props.$radioInnerSize)};
+    width: ${(props) => parseSpacing(props.$radioInnerSize)};
     background: ${parseColor("black")};
-    lay: block;
     position: absolute;
     border-radius: 50%;
     border: ${parseBorder("border-solid-m")} ${parseColor("white")};
   }
-      
 `;
 
 // This is a hack to force React to rerender when the disabled prop is changed. Otherwise the pseudo element is not updated.
 const DisabledVisibleRadioButtonInput = styled(VisibleRadioButtonInput)`
   ${HiddenRadioButtonInput}:checked ~ &::after {
     content: "";
-    height: ${parseSpacing("all-spacing-4")};
-    width: ${parseSpacing("all-spacing-4")};
+    height: ${(props) => parseSpacing(props.$radioInnerSize)};
+    width: ${(props) => parseSpacing(props.$radioInnerSize)};
     background: ${parseColor("bg-btn-primary-disabled")};
-    lay: block;
     position: absolute;
     border-radius: 50%;
     border: ${parseBorder("border-solid-m")} ${parseColor("white")};
   }
 `;
 
-type OakRadioButtonProps = {
+export type OakRadioButtonProps = {
   id: string;
-  label: string;
+  label: ReactNode;
```

Comment:
Making `label` a `ReactNode` allows us to pass any content into the  label "slot". This allows us to insert the image from `OakQuizRadioButton` as part of the label